packages/terra-dropdown-button/src/terra-dev-site/test/dropdown-button/BlockSplitButton.test.jsx

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-console */
+import React from 'react';
+
+// eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
+import SplitButton, { Button } from '../../../SplitButton';
+
+export default () => (
+  <SplitButton
+    primaryOptionLabel="Primary option"
+    onClick={() => { console.log('primary option'); }}
+    isBlock
+  >
+    <Button label="1st Option" onClick={() => { console.log('hi'); }} />
```

Comment:
Let's update this example to write content to the page instead of a console.log.